react/week-19/lesson.md

```diff
@@ -399,55 +403,53 @@ The convention is to name component files the exactly same as the component (inc
 
 ## Making an Argument for Props
 
-What's the problem with our `HelloMentor` component above? Hint: imagine what our boss might ask for with this small application. What could our boss ask for which would mean we would have to make changes to the code?
```

Comment:
I think this a good call to remove this from the syllabus text (I can get a bit wordy, sorry!).

However I have found that it is useful to contextualise examples in a "real world" example like a job. I've observed that our students don't really have a good idea of what a development job looks like on a daily basis. Maybe it's worth giving a story like this in the class?

Reply:
I think adding to the mentor notes is good - I've found that having concrete examples is good reference for new mentors